src/util/server/getMainPageText.ts

```diff
@@ -0,0 +1,17 @@
+import { promises as fs } from 'fs';
+import path from 'path';
+import { is } from 'typescript-is';
+
+import type { MainPageText } from 'custom-types';
+
+export const getMainPageText = async (): Promise<MainPageText> => {
+  const mainPageTextPath = path.join(process.cwd(), 'config', 'main-page-text.json');
+  const mainPageTextUnparsed = await fs.readFile(mainPageTextPath, 'utf8');
+  const mainPageText = JSON.parse(mainPageTextUnparsed) as MainPageText;
+
+  if (!is<MainPageText>(mainPageText)) {
+    throw TypeError('The main page text does not fit the schema');
+  }
```

Comment:
👍 

can we do this for the other json file and obj in config dir

Reply:
Sadly I need to do a prebuild file for the other config file because I cannot use the `is` function outside of the `getStaticProps` and `getStaticPaths` functions.